buildSrc/subprojects/profiling/src/main/kotlin/org/gradle/gradlebuild/profiling/buildscan/BuildScanPlugin.kt

```diff
@@ -188,48 +185,18 @@ open class BuildScanPlugin : Plugin<Project> {
     fun isEc2Agent() = java.net.InetAddress.getLocalHost().hostName.startsWith("ip-")
 
     private
-    fun Project.extractCiOrLocalData() {
+    fun Project.extractCIData() {
         if (isCiServer) {
             buildScan {
-                tag("CI")
+                setCompileAllScanSearch(execAndGetStdout("git", "rev-parse", "--verify", "HEAD"))
```

Comment:
Are we running that twice now? Once for the plugin and once here? Wouldn't it be nicer to expose the commit id from the plugin, so it can be used when building on top of it?

Reply:
Yes, the command is run twice. We can probably store the information in the plugin.

Reply:
This should also happen in the background, if not it will block execution of the project.

Reply:
Done.